java/tests/com/google/privacy/differentialprivacy/GaussianNoiseConfidenceIntervalTest.java

```diff
@@ -0,0 +1,727 @@
+package com.google.privacy.differentialprivacy;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import static com.google.common.truth.Truth.assertWithMessage;
+import static org.junit.Assert.assertThrows;
+
+@RunWith(JUnit4.class)
+public class GaussianNoiseConfidenceIntervalTest {
+  private static final Noise NOISE = new GaussianNoise();
+  private static final double TOLERANCE = 1e-7;
+  private static final double DEFAULT_EPSILON = 0.5;
+  private static final int DEFAULT_L_0_SENSITIVITY = 1;
+  private static final double DEFAULT_L_INF_SENSITIVITY = 1.0;
+  private static final double DEFAULT_DELTA = 0.3;
+  private static final double DEFAULT_NOISED_X = 70;
+  private static final double DEFAULT_ALPHA = 0.1;
+
+  private boolean approxEqual(double a, double b) {
+    double mxMagnitude = Math.max(Math.abs(a), Math.abs(b));
+    return Math.abs(a - b) <= TOLERANCE * mxMagnitude;
+  }
+
+  private void isEqual(ConfidenceInterval a, ConfidenceInterval b) {
```

Comment:
we don't need this, we decided to use isApproxEqual for all cases.
Also rename isApproxEqual to verifyApproxEqual